draft-ietf-taps-impl.md

```diff
@@ -489,7 +489,7 @@ Instead, calling the ConnectionReceived event may be delayed until the first Mes
 
 ## Handling racing with "unconnected" protocols {#unconnected-racing}
 
-While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as a local route to the peer endpoint is confirmed.
+While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as connectivity with the peer endpoint is established.
```

Comment:
```suggestion
While protocols that use an explicit handshake to validate a Connection to a peer can be used for racing multiple establishment attempts in parallel, "unconnected" protocols such as raw UDP do not offer a way to validate the presence of a peer or the usability of a Connection without application feedback. An implementation should consider such a protocol stack to be established as soon as the Transport Services system has selected a valid path.
```

Reply:
Tommy's text is an improvement, but I wonder if we should be more brutally obvious. Perhaps "the Transport Services system has selected a destination IP address to which it has a route."

That text is unambiguous, with no wiggle room about what is "valid", etc.

Reply:
I understand the problem now. I like Tommy's text, and I like your suggestion even better. Let's see how many thumbs up this gets...  if we get the critical mass, maybe we can just apply this fix and be done with this.

Reply:
Close: I don't that much like "route" - I'd prefer to not mention the routing system. So I suggest:
"the Transport Services system has selected a  a valid path and the corresponding destination IP address it will use."

Reply:
I think @martinduke was trying to get rid of "valid path", I agree that term is not very clear.

Reply:
ah - but valid "route" is much worse. "established" connection is yucky. Maybe "A path on which it is valid to send" ...not implying the path is valid end-to-end?